less/admin/PermissionsPage.less

```diff
@@ -125,6 +143,13 @@
   th {
     font-weight: normal;
   }
+
+  &:not(:hover) {
+    th {
+      background: @body-bg;
+    }
+  }
```

Comment:
It would cleaner to instead just add the normal bg color to the actual element `PermissionGrid-child` and not the th so that we don't have to use the `:not` here.

Additionally I'd also like to see that same bg color applied to `.PermissionGrid-section` so that the hidden tag names don't weirdly show up at times when scrolling down.

Reply:
> It would cleaner to instead just add the normal bg color to the actual element PermissionGrid-child and not the th so that we don't have to use the :not here

Oooops, obviously!